### PR TITLE
Updated definition of hasMember

### PR DESCRIPTION
**Pull request by uscholdm**
Updated definition of hasMember

### CODE REVIEW

File: docs/release_notes/439-hasMember-definition.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+### Patch Updates
+Issue [#439](https://github.com/semanticarts/gist/issues/439). Changes:
+  
+- Updated definition of hasMember
```
Comment (rjyounes) [inline docs/release_notes/439-hasMember-definition.md:4]:
```suggestion
- Updated definition of `gist:hasMember` 
```

File: gistCore.ttl [hunk 2]
```diff
@@ -3170,7 +3170,7 @@ gist:hasMagnitude
 
 gist:hasMember
 	a owl:ObjectProperty ;
-	skos:definition "Relates a Collection to its member individuals."^^xsd:string ;
+	skos:definition "Relates something, such as a collection or organization, to its member individuals."^^xsd:string ;
 	skos:prefLabel "has member"^^xsd:string ;
 	gist:domainIncludes gist:Collection ;
```
Comment (rjyounes) [inline gistCore.ttl:3175]:
```suggestion
	gist:domainIncludes gist:Collection , gist:Organization
```

File: docs/release_notes/439-hasMember-definition.md (hunk 1 above)
Comment (rjyounes) [inline docs/release_notes/439-hasMember-definition.md:4]:
```suggestion
- Updated definition of `gist:hasMember`. 
```

**Review by rjyounes — CHANGES_REQUESTED** [review]

File: docs/release_notes/439-hasMember-definition.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+### Patch Updates
+Issue [#439](https://github.com/semanticarts/gist/issues/439). Changes:
+  
+- Updated definition of `gist:hasMember`.
```
Comment (rjyounes) [inline docs/release_notes/439-hasMember-definition.md:4]:
```suggestion
- Updated definition of `gist:hasMember`. Issue [#439](https://github.com/semanticarts/gist/issues/439). 
```

Similar comment to the one I made on PR #995.

**Review by rjyounes — CHANGES_REQUESTED** [review]
Change looks good, just a question about the release note.

**Review by uscholdm — COMMENTED** [review]
Made recommended changes.

**Review by rjyounes — APPROVED** [review]

**Pull request merged**